Misc/NEWS.d/next/Core and Builtins/2024-01-15-18-11-48.gh-issue-113190.OwQX64.rst

```diff
@@ -0,0 +1,20 @@
+This updates the interned string deallocation function
+``_PyUnicode_ClearInterned`` to delete all interned strings during runtime
+finalization when calling ``Py_Finalize``, regardless of their reference
+count.
+
+Worth noting that if an extension accidentally holds onto a interned string,
+event after calling Py_Finalize, it will result in use-after-free error,
```

Comment:
Should be "even after calling ...", I think.

Reply:
Good catch, fixed!